### ISSUE TITLE
Typo in trainer?

### ISSUE THREAD

**Issue by Gaaaavin** [NetoPedro/FocusFace#3]
In line 81 of `trainer.py`, there is `mask*0`. However if I understand the pipeline correctly, it should be `labels2*0` instead of masks. Is this a typo?

**Comment by NetoPedro** [comment]
Yes @Gaaaavin, you are correct. That is a typo. You can submit a pull request if you want :)
